recipes/cf-nvidia-tools/recipe.yaml

```diff
@@ -0,0 +1,46 @@
+context:
+  name: cf-nvidia-tools
+  version: "1.0.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - path: bin
+    target_directory: bin
+
+build:
+  skip:
+    - not linux
+  noarch: generic
+  script:
+    - ls -lah
+    - cp -r bin $PREFIX/bin
+  number: 0
+
+requirements:
+  run:
+    - __unix
+    - patchelf >=0.12
+
+tests:
+  - script:
+    - export c_stdlib_version="2.17.0"
+    - check-glibc ${CONDA_PREFIX}/lib/*.so.*
+
+about:
+  summary: NVIDIA's feedstock validation and linting tools for conda-forge
+  description: >
+    This package contains CLI tools for validating and linting NVIDIA's conda recipes on
+    conda-forge. For a description of the tools see the README in the package feedstock. The
+    tools are hosted directly in the feedstock; there is no external source code repository
+    for these tools at this time.
```

Comment:
Do we want this to be one long string or do want to have it wrapped a certain way?

FWIW find this website to be a useful resource when playing with YAML string formatting: https://yaml-multiline.info

Reply:
I want one long string. anaconda.org handles the wrapping of long lines.